doc/api/crypto.md

```diff
@@ -1706,6 +1706,71 @@ This should normally never take longer than a few milliseconds. The only time
 when generating the random bytes may conceivably block for a longer period of
 time is right after boot, when the whole system is still low on entropy.
 
+### crypto.randomFillSync(buf[, offset][, size])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `buf` {Buffer}
+* `offset` {number}
+* `size` {number}
+
+This function is similar to [`crypto.randomBytes()`][] but requires the first
+argument to be a [`Buffer`][] that will be filled as well as returned. Another
+significant difference between the two is that this function will always be
+synchronous. To use the asynchronous form, please see
+[`crypto.randomFill()`][]. The `offset` argument defaults to `0`.
```

Comment:
That's a lot of words and doesn't doc the params in the parm section, but in the long paragraph. I suggest replacing params section and the descriptive paragraph with:
```
* `buf` {Buffer} Must be supplied. This buffer is returned by `randomFillSync()`.
* `offset` {number} Defaults to `0`.
* `size` {number} Defaults to `buf.length  - offset`.

Synchronous version of [`crypto.randomFill()`][].
```